apps/docs/content/guides/auth/third-party/auth0.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: 'auth-third-party-auth0'
+title: 'Auth0'
+subtitle: 'Use Auth0 with your Supabase project'
+---
+
+Auth0 can be used as a third-party authentication provider alongside Supabase Auth, or standalone, with your Supabase project.
+
+## Getting started
+
+1. First you need to add an integration to connect your Supabase project with your Auth0 tenant. You will need your tenant ID (and in some cases region ID).
+2. Add a new Third-party Auth integration in your project's [Authentication settings](/dashboard/project/_/settings/auth).
+3. Assign the `role: 'authenticated'` custom claim to all JWTs by using an Auth0 Action.
```

Comment:
```suggestion
3. Assign the `role: 'authenticated'` custom claim to all JWTs by using an [Auth0 Action](https://auth0.com/docs/get-started/apis/scopes/sample-use-cases-scopes-and-claims#add-custom-claims-to-a-token).
```

Can consider linking to Auth0 docs but at the same time we'll have to update if their docs change. And also would be reasonable to assume that folks looking at this page are familiar with Auth0

Reply:
nvm it's linked below. I'll finish reading then comment.